src/devices/Ccs811/Ccs811Sensor.cs

```diff
@@ -0,0 +1,518 @@
+                return false;
+            }
+
+            equivalentCO2InPpm = BinaryPrimitives.ReadInt16BigEndian(toRead.Slice(0, 2));
+            equivalentTotalVolatilOrganicCompoundInPpb = BinaryPrimitives.ReadInt16BigEndian(toRead.Slice(2, 2));
+            rawCurrentSelected = toRead[6] >> 2;
+            rawAdcReading = ((toRead[6] & 0b0000_0011) << 2) + toRead[7];
+            return ((equivalentCO2InPpm >= 400) && (equivalentCO2InPpm <= 29206) && (equivalentTotalVolatilOrganicCompoundInPpb >= 0) && (equivalentTotalVolatilOrganicCompoundInPpb <= 32768));
+        }
+
+        /// <summary>
+        /// Read the equivalent CO2 in ppm and equivalent Total Volatile Compound in ppb
+        /// </summary>
+        /// <param name="equivalentCO2InPpm">The equivalent CO2 (eCO2) output range for CCS811 is from
+        /// 400ppm up to 29206ppm.</param>
+        /// <param name="equivalentTotalVolatilOrganicCompoundInPpb">The equivalent Total Volatile Organic Compound (eTVOC)
+        /// output range for CCS811 is from 0ppb up to 32768ppb</param>
+        /// <returns>True if success</returns>
+        public bool ReadGasData(out int equivalentCO2InPpm, out int equivalentTotalVolatilOrganicCompoundInPpb)
+        {
+            return ReadGasData(out equivalentCO2InPpm, out equivalentTotalVolatilOrganicCompoundInPpb, out int curr, out int adc);
+        }
+
+        /// <summary>
+        /// Get or set the encoded version of the current baseline used in Algorithm Calculations
+        /// </summary>
+        /// <remarks>A previously stored value may be written back to this two byte
+        /// register and the Algorithms will use the new value in its
+        /// calculations(until it adjusts it as part of its internal Automatic
+        /// Baseline Correction). Please refer to documentation to understand when to restore a
+        /// previous baseline: https://www.sciosense.com/wp-content/uploads/2020/01/Application-Note-Baseline-Save-and-Restore-on-CCS811.pdf</remarks>
+        public ushort BaselineAlgorithmCalculation
+        {
+            get
+            {
+                Span<byte> baseline = stackalloc byte[2];
+                ReadRegister(Register.BASELINE, baseline);
+                return BinaryPrimitives.ReadUInt16BigEndian(baseline);
+            }
+
+            set
+            {
+                Span<byte> baseline = stackalloc byte[2];
+                BinaryPrimitives.WriteUInt16BigEndian(baseline, value);
+                WriteRegister(Register.BASELINE, baseline);
+            }
+        }
+
+        /// <summary>
+        /// Set the environmental data, this is impacting the equivalent calculation
+        /// of the gas.
+        /// </summary>
+        /// <param name="temperatureCelsius">The temperature in Celsius</param>
+        /// <param name="humidityPerCent">The relative humidity from 0 to 100%</param>
+        public void SetEnvironmentData(double temperatureCelsius, double humidityPerCent)
```

Comment:
I believe there is a unit for percentage (Ratio), if not then please at least fix typo (casing: `humidityPercent`)

Reply:
Humidity goes from 0.0 to 100.0, should I adjust the name? or it ```humidityPercent``` the correct one?
OK for the Temperature unit, will adjust as well

Reply:
We use Ratio for relative humidty elsewhere: https://github.com/dotnet/iot/blob/master/src/devices/Bmxx80/Bme280.cs#L85
other options are:
- `double humidityRatio` (0.0-1.0)
- `double humidityPercent` (0.0-100.0)

Ratio is preferred for consistency